add change log and bump version to 0.1.11 (#350)
streamnative/function-mesh: README.md

- [pulsar-cluster](https://pulsar.apache.org/docs/en/pulsar-2.0/)
- [pulsar-functions](https://pulsar.apache.org/docs/en/functions-overview/)

## Compatibility

### Kubernetes compatibility matrix

> The following Kubernetes versions are supported and work as we test against these versions in their respective branches. But note that other versions might work as well.

Reviewer: ```suggestion
This table outlines the supported Kubernetes versions. We have tested these versions in their respective branches. But note that other versions might work as well.
```
